src/bin/cargo-ziggy/fuzz.rs

```diff
@@ -295,8 +297,8 @@ impl Fuzz {
                     _ => process::Stdio::null(),
                 };
                 let final_sync = match job_num {
-                    0 => "1",
-                    _ => "0",
+                    0 => "AFL_FINAL_SYNC",
+                    _ => "_DUMMY_VAR",
```

Comment:
I know this chance looks weird. the thing is afl-fuzz only cares for a boolean env var if it is set or not. so setting "0" is equal to "1" or "foobar".

Reply:
Hahaha good to know